### PR TITLE
Concept exercise for "Namespaces" (closes #609)

### LINKED ISSUE #609: Concept Exercise for "Namespaces"
Write an exercise for Namespaces, that has only "Basics" as a prerequisite.

### PR DESCRIPTION
**Pull request by vaeng**

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
@vaeng I'll look at this next week, I'm terribly busy right now

**Comment by vaeng** [comment]
> @vaeng I'll look at this next week, I'm terribly busy right now

Thanks for the info. It's not super busy. Take care :)

File: exercises/concept/last-will/.docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
+
+## 2. Define a function that assembles the secret account number
+
+Define the `assemble_account_number(int secret_modifier)` function that takes the secret modifier as an argument and returns the assembled account number as an `int`.
+
+To get the correct number, you have to sum up the `bank_number_part` from each of the three families.
+
+## 3. Define a function that assembles the secret code
+Define the `assemble_code()` function that returns the resulting code by combining the fragments from the three families.
```
Comment (ErikSchierboom) [inline exercises/concept/last-will/.docs/instructions.md:31]:
```suggestion

Define the `assemble_code()` function that returns the resulting code by combining the fragments from the three families.
```

File: exercises/concept/last-will/.docs/instructions.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+You work for a prestigious law firm that is specialized in handling unique testament requests.
+
+In this exercise, you are going to open a mysterious vault.
+You are the executor of the estate and will assemble the long-kept secret codes of three families to get an account number and the matching code.
+
+To prevent any family from opening the vault alone, it can only be opened by combining their knowledge with a secret modifier that you know from the last will.
+
+You have three tasks, all related to helping the families to open the vault.
+
+## 1. Define a Namespace for the executor of estate functions
```
Comment (ErikSchierboom) [inline exercises/concept/last-will/.docs/instructions.md:12]: Tiny nit: these task headings are very technical. In general, we try to make these more like story-based tasks. E.g. https://github.com/exercism/fsharp/blob/main/exercises/concept/bandwagoner/.docs/instructions.md#2-create-a-teams-coach lists the task as "Create a team's coach", and not as "Create a coach record". But if this rephrasing is hard for this exercise, it's totally fine.

File: exercises/concept/last-will/.docs/instructions.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Instructions
+
+You work for a prestigious law firm that is specialized in handling unique testament requests.
+
+In this exercise, you are going to open a mysterious vault.
```
Comment (ErikSchierboom) [inline exercises/concept/last-will/.docs/instructions.md:5]: I like the story!

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/concept/last-will/.docs/instructions.md (hunk 2 above)
Comment (vaeng) [reply exercises/concept/last-will/.docs/instructions.md]: I think I made it more story-like now.

**Pull request merged**